### PR TITLE
Add start_time to system.runtime.nodes

### LINKED ISSUE #3406: Add start_time to system.runtime.nodes
This should be a `timestamp` of when the node started. It can be added to the announcement as a property like `node_version` in `ServerMainModule`, then pulled out in `DiscoveryNodeManager` and added to `PrestoNode`.

### PR DESCRIPTION
**Pull request by yaoxin226**
Fix #3406

### CODE REVIEW

**Comment by yaoxin226** [comment]
@electrum Could you take a look when you have time?

**Comment by haozhun** [comment]
@yaoxin226 Are you still working on this?

**Comment by yaoxin226** [comment]
haven't work on it for a long time. I would love to start working on it if this PR still make sense now.

**Comment by yaoxin226** [comment]
@martint @haozhun Addressed the comments. Could you take a look?

File: presto-main/src/main/java/com/facebook/presto/connector/system/NodeSystemTable.java [hunk 1]
```diff
@@ -73,14 +79,19 @@ public RecordCursor cursor(ConnectorSession session, TupleDomain<Integer> constr
...
         for (Node node : allNodes.getInactiveNodes()) {
-            table.addRow(node.getNodeIdentifier(), node.getHttpUri().toString(), getNodeVersion(node), isCoordinator(node), Boolean.FALSE);
+            table.addRow(node.getNodeIdentifier(), node.getHttpUri().toString(), getNodeVersion(node), getNodeStartTime(node), isCoordinator(node), Boolean.FALSE);
         }
         return table.build().cursor();
     }
 
+    private static long getNodeStartTime(Node node)
+    {
+        return packDateTimeWithZone(node.getStartTime().toEpochMilli(), TimeZoneKey.getTimeZoneKey(TimeZone.getDefault().toZoneId().toString()));
```
Comment (haozhun) [inline presto-main/src/main/java/com/facebook/presto/connector/system/NodeSystemTable.java]: You can use `session.getTimeZoneKey()` to display it in the client's timezone.

**Comment by yaoxin226** [comment]
@haozhun fixed your comment and rebased on current master

**Comment by raghavsethi** [comment]
Closing this due to lack of response. Please rebase and open another PR if this is still required.

**Pull request closed without merging**